.config/webpack/webpack.config.ts

```diff
@@ -81,7 +81,7 @@ const config = async (env): Promise<Configuration> => ({
           loader: 'swc-loader',
           options: {
             jsc: {
-              baseUrl: './src',
+              baseUrl: path.resolve(process.cwd(), SOURCE_DIR),
```

Comment:
The reason they don't want changes here is because these files get autogenerated and overwritten by the update command: https://www.npmjs.com/package/@grafana/create-plugin#update-your-plugin-build-config

So even if it's a pain, I think it's probably worth putting the customization in the other webpack.config file